Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt

```diff
@@ -18,40 +19,35 @@ object DiaryData {
     val DATA_ITEMS = listOf(
         ListItem.Data(
             R.drawable.ic_contact_diary_person_item,
-            "Max Mustermann",
-            null,
-            listOf(
-                R.string.contact_diary_person_encounter_duration_below_15_min,
-                R.string.contact_diary_person_encounter_mask_with,
-                R.string.contact_diary_person_encounter_environment_inside
-            ),
-            "Notizen notizen",
-            ListItem.Type.PERSON
+            "Rewe",
+            Duration.standardMinutes(30),
+            attributes = null,
+            circumstances = null,
+            ListItem.Type.LOCATION
         ),
-
         ListItem.Data(
             R.drawable.ic_contact_diary_person_item,
-            "Erika Mustermann",
+            "Andrea Steinhauer",
             null,
             listOf(
-                R.string.contact_diary_person_encounter_environment_inside
+                R.string.contact_diary_person_encounter_duration_below_15_min,
+                R.string.contact_diary_person_encounter_environment_outside
             ),
-            "Notizen notizen",
+            null,
             ListItem.Type.PERSON
         ),
 
         ListItem.Data(
```

Comment:
UX would like us to remove "Bäcker" entirely

Reply:
Done, see new screenshot: 

![ContactDiaryOverviewFragment_2](https://user-images.githubusercontent.com/10398034/109298417-dbba0a80-7833-11eb-82b0-69a65430717d.png)